index.js

```diff
@@ -31,6 +32,9 @@ const copyTextToClipboard = input => {
 		isSuccess = document.execCommand('copy');
 	} catch (_) {}
 
+	// Get the focus back on previously focused element
+	previouslyFocusedElement.focus();
```

Comment:
If there's no previously focused element, this will fail. I would also move it to right before the return statement to ensure nothing else will affect the focus state.

Reply:
Right, makes sense. Updated the patch.